dev-docs/FrequentTasks.md

```diff
@@ -331,3 +331,34 @@ knife ssl fetch
 knife ssl check
 ```
 
+## Using S3 instead of Bookshelf
+
+If you would like to configure chef-server to use S3 instead of Bookshelf, you can
+type the following into the bash shell of your chef-server.  This assumes you are
+using a root login.  Make obvious substitutions as needed, e.g. vip, external_url,
```

Comment:
The variables to be substituted are in chef-server.rb ?

Reply:
Yes.  See bash comment below `# copies this into the bottom of /etc/opscode/chef-server.rb`

Reply:
It's a script you can copy into the bash shell of your running chef-server to make the changes you need to `chef-server.rb`, `environment`, etc.